chore(docs): adding a faq to the docs (#2053)
winglang/wing: docs/07-faq/022-supported-provisioning-engines.md

---
title: Which provisioning engines are supported by Wing?
id: supported-provisioning-engines
keywords: [faq, supported provisioning engines, winglang, Wing programming language, Wing language, Terraform, AWS CDK, Pulumi]
---

> Wing was built from the ground up to make it an ideal choice for building applications on any cloud, using a varaity of provisioning engines.
> Since wing is in early stages of developemnt, it officially only supports the Terraform provisioning engine at the moment. AWS CDK is supported by the community.

Reviewer: ```suggestion
Wing was built from the ground up to make it an ideal choice for building applications on any cloud, using a varaity of provisioning engines.
Since Wing is in early stages of development, it officially only supports the Terraform provisioning engine. AWS CDK (based on CloudFormation) is also supported by the community.
```

> Have other provisioning engines you would like to use with Wing? Let us know by leaving a comment [here](link)!

Reviewer: oh, we should create an issue to link to for this 😅

> As wing matures, we will add official support for AWS CDK, as well as other provisioning engines, such as Kubernetes, ARM, and more.

Reviewer: ```suggestion
Have other provisioning engines you would like to use with Wing? Let us know by leaving a comment [here](link)!
```
